services/transactionService.ts

```diff
@@ -123,6 +124,23 @@ const transactionsWithPaybuttonsAndPrices = Prisma.validator<Prisma.TransactionD
 
 export type TransactionsWithPaybuttonsAndPrices = Prisma.TransactionGetPayload<typeof transactionsWithPaybuttonsAndPrices>
 
+export const transactionWithAddressAndPricesAndButtons = Prisma.validator<Prisma.TransactionDefaultArgs>()({
```

Comment:
this won't be used anywhere externally, it's only needed to create the type below. No need to export it.

Reply:
Actually, check the comment below, probably this shouldn't exist together with the type it is used for.